src/test/java/org/broadinstitute/hellbender/tools/walkers/vqsr/NeuralNetStreamingExecutorIntegrationTest.java

```diff
@@ -0,0 +1,41 @@
+package org.broadinstitute.hellbender.tools.walkers.vqsr;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
+import org.broadinstitute.hellbender.utils.test.IntegrationTestSpec;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+/**
+ * Integration tests for {@link NeuralNetStreamingExecutor}.
+ * Created by sam on 1/8/18.
+ */
+public class NeuralNetStreamingExecutorIntegrationTest extends CommandLineProgramTest {
+    private static String architectureHD5 = packageMainResourcesDir + "tools/walkers/vqsr/cnn_1d_annotations.hd5";
+    private static final String space = " ";
+    /**
+     * Run the tool on sample VCF.
+     */
+    @Test(groups = {"python"})
+    public void testInference() {
+        final String inputVCF = largeFileTestDir + "VQSR/recalibrated_chr20_start.vcf";
+        final ArgumentsBuilder argsBuilder = new ArgumentsBuilder();
+        argsBuilder.addArgument(StandardArgumentDefinitions.VARIANT_LONG_NAME, inputVCF)
+                .addArgument(StandardArgumentDefinitions.OUTPUT_LONG_NAME, "%s")
+                .addArgument(StandardArgumentDefinitions.REFERENCE_LONG_NAME, b37_reference_20_21)
+                .addArgument("architecture", architectureHD5)
+                .addArgument(StandardArgumentDefinitions.ADD_OUTPUT_VCF_COMMANDLINE, "false");
+
+        runCommandLine(argsBuilder);
```

Comment:
Tests should be written using either ArgumentsBuilder/runCommandLine, or IntegrationTestSpec, but not both. IntegrationTestSpec is "old-style", but convenient. It automatically generates a temporary output file and substitutes the name for %s, and also compares the output to the expected results file. If you use runCommandLine, you need to generate your own temporary output file, and do your own expected results comparison. Since this method mixes both styles, it runs the test twice  - first using runCommandLine with an output filename of literal "%s",  and then again via executeTest using a generated temp file. Take a look at [SelectVariantsIntegrationTest](https://github.com/broadinstitute/gatk/blob/f26cb5dbd7f8bb4ee549c7c1c1044e8ec7a28645/src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariantsIntegrationTest.java#L27) as an example of using InegrationTestSpec, or [PrintReadSparkIntegrationTest](https://github.com/broadinstitute/gatk/blob/054f7481b5fff4b781de7aa74433f7fd685ea2eb/src/test/java/org/broadinstitute/hellbender/tools/spark/pipelines/PrintReadsSparkIntegrationTest.java#L111) for the other style.

Reply:
fixed